src/AS_02_IAB.cpp

```diff
@@ -0,0 +1,617 @@
+/*
+Copyright (c) 2011-2020, Robert Scheler, Heiko Sparenberg Fraunhofer IIS,
+John Hurst, Pierre-Anthony Lemieux
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions
+are met:
+1. Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+2. Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in the
+   documentation and/or other materials provided with the distribution.
+3. The name of the author may not be used to endorse or promote products
+   derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
+IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+#include "AS_02_internal.h"
+#include "AS_02_IAB.h"
+
+#include <iostream>
+#include <iomanip>
+#include <array>
+
+namespace Kumu {
+  class RuntimeError : public std::runtime_error {
```

Comment:
On macOS Catalina, clang-1100.0.33.17, the build fails here:
```
/Users/wolfgangruppel/src/asdcplib-sandflow/asdcplib-sandflow/src/AS_02_IAB.cpp:38:9: error: exception specification of overriding function is more lax than base version
  class RuntimeError : public std::runtime_error {
        ^
```

Adding `-std=c++11` as compiler option solves the problem, but enforcing C++11 might cause issues on other platforms.

Reply:
The objective is compatibility with C++98, so I plan to fix it.

What about setting `set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 98)` in `src/CMakeList.txt`?

Reply:
@IMFTool Addressed at https://github.com/cinecert/asdcplib/pull/33/commits/8be2ad09b94fbf2f023e24cf1b7eec10c03047b4

Would you like to propose a change to `CMakeList.txt`?

Reply:
That fixes the compiler error. No need to Change `CMakeLists.txt` IMHO.

Reply:
@IMFTool The error is triggered on Ubuntu and Win10 only when `CMakeLists.txt` explicitly constrain the compiler to C++98.

Reply:
By default, clang compiles according to C++98 [1], while Linux gcc and VS by default compile according do C++ >=11.
With the [latest patch](https://github.com/cinecert/asdcplib/commit/8be2ad09b94fbf2f023e24cf1b7eec10c03047b4), the PR build fine on macOS with default clang setting (i.e. C++98).
[1] [https://clang.llvm.org/cxx_status.html](https://clang.llvm.org/cxx_status.html)

Reply:
Tossing a 
```#include <stdexcept>```
up in src/AS_02_IAB.cpp fixed that same build issue for me on one of my CentOS boxes. It built fine as-is on a Fedora one.

Reply:
Addressed at https://github.com/cinecert/asdcplib/pull/33#discussion_r421689866